axelor-base/src/main/resources/views/PriceList.xml

```diff
@@ -44,6 +44,7 @@
 			<panel-related name="historizedPriceListPanel" field="historizedPriceList" colSpan="12" readonly="true" form-view="price-list-form" grid-view="price-list-hist-grid"/>
 			<button name="historizePriceListBtn" hideIf="!id" title="Historize Price List" onClick="save,com.axelor.apps.base.web.PriceListController:historizePriceList"/>
 		</panel>
+		<panel-dashlet name="PartnerPriceListPanel" title="Partner price lists" action="action-price-list-view-dashlet-partner-price-list" colSpan="12"/>
```

Comment:
panel name should be 'partnerPriceListPanel' not 'PartnerPriceListPanel' (starts with small case letter)